x-pack/plugin/sql/qa/server/src/main/resources/unsigned-long.sql-spec

```diff
@@ -13,61 +13,61 @@
 plus
 SELECT 18446744073709551614 + 1 AS x;
 plusLongMax
-SELECT 9223372036854775808 + ROUND(RAND(SECOND(CURRENT_TIMESTAMP())) * 10000) AS x;
+SELECT 9223372036854775808 + ROUND(RAND(DAY(CURRENT_TIMESTAMP())) * 10000) AS x;
 minus
-SELECT 18446744073709551615 - ROUND(RAND(SECOND(CURRENT_TIMESTAMP())) * 10000) AS x;
+SELECT 18446744073709551615 - ROUND(RAND(DAY(CURRENT_TIMESTAMP())) * 10000) AS x;
 divide
-SELECT 18446744073709551615 / ROUND(RAND(SECOND(CURRENT_TIMESTAMP())) * 10000) AS x;
+SELECT 18446744073709551615 / ROUND(RAND(DAY(CURRENT_TIMESTAMP())) * 10000) AS x;
 multiply
 SELECT 18446744073709551615 * 1 AS x;
 mod
-SELECT 18446744073709551615 % ROUND(RAND(SECOND(CURRENT_TIMESTAMP())) * 10000) AS x;
+SELECT 18446744073709551615 % ROUND(RAND(DAY(CURRENT_TIMESTAMP())) * 10000) AS x;
 
 abs
-SELECT ABS(9223372036854775808 + ROUND(RAND(SECOND(CURRENT_TIMESTAMP())) * 10000)) AS x;
+SELECT ABS(9223372036854775808 + ROUND(RAND(DAY(CURRENT_TIMESTAMP())) * 10000)) AS x;
 // ceil
-// SELECT CEIL(9223372036854775808 + RAND(SECOND(CURRENT_TIMESTAMP())) * 10000) AS x;
+// SELECT CEIL(9223372036854775808 + RAND(DAY(CURRENT_TIMESTAMP())) * 10000) AS x;
```

Comment:
not really related to this PR but are the commented out on purpose?

Reply:
There's this comment in the header:
```
ES's CEIL & FLOOR don't work with unsigned_longs (TODO) + they can return longs, while H2's always returns doubles
```

Reply:
> did this already trigger any CI failures?

forgot to answer: yes, it did a couple.